### DISCUSSION TITLE
Default out of place HUD elements

### DISCUSSION THREAD

**Discussion by thAlbino60** [KdGaming0/SkyBlock-Enhanced-Modpack#9] [Help] [answered]
Hello!

So, I am using Prism launcher and I just installed the latest version of the modpack directly from the launcher. I also already inserted the recommended Optifine version into the mods' folder. My monitor is a 1920x1080p and I didn't change any configurations from any of the mods nor from Minecraft. But still, some of my HUD elements seem that they are out of place, for instance the Chocolate Factory layout that is behind the inventory menu. Should that be happening?

![2024-11-18_20 26 31](https://github.com/user-attachments/assets/1ce69603-554a-4094-a2d7-5049ca82391e)

**Comment by KdGaming0** [comment, marked as answer]
The HUD elements are just a starting point for you to tweak and customize however you like. The goal is to have a solid setup right out of the box—no overlapping and with most stuff in good spots.

I noticed the Chocolate Factory layout is one thing I forgot to fix, but I’ll sort that out in the next update. Those GUI elements are from the Skyhanni mod, and you can easily move them by pressing G and dragging them wherever you want.
You also say that you have some other hud elements that are out of place it would be greatly appreciated if you let me know so I can fix those too. Going through every HUD element for each mod is a lot of work, so I’ve most likely missed some of them.

I’m planning a big update where I’ll do a full revamp of the HUD elements, but I haven’t found the time or motivation to do that just yet.

Thanks for pointing that out and using my modpack!

**Reply by thAlbino60** [reply]
Oh, I understand what you say! Thank you for letting me know how to edit Skyhanni's GUI elements.

I really understand it can be quite demotivating to do all that work with mods. In fact that's why I decided to search for a modpack to start. Having multiple mods that do the same thing just and having to tune a lot of settings between all of them was just a lot for me and it was actually making me don't play the game! But I just want to let you know that your modpack really helped me a lot, and I'm sure a lot of people too! Finding a modpack like this made my life so easier, and your job is so well done with this!! I just wanted to say thank you for all your efforts and work you putted into this :) . I'm seriously going to recommend to some of my friends who, as myself, don't want to configurate all those mods themselves.

Also, sure, I will tell you what are the other GUI elements out of place! I'm also out of time, but as soon as possible I publish another discussion.
